renovate.json

```
@@ -12,6 +12,14 @@
     {
       "packageNames": ["@octokit/rest", "@slack/web-api", "googleapis"],
       "reviewers": ["beyang"]
+    },
+    {
+      "packageNames": ["^@visx/"],
```

Since this is a regex, I think it needs to be `packagePatterns` else it wouldn't match
```suggestion
      "packagePatterns": ["^@visx/"],
```